test/claim-ts.ts

```diff
@@ -0,0 +1,202 @@
+const { expect } = require("chai");
+import { BigNumber } from "@ethersproject/bignumber";
+import { claim } from "../ts/claim";
+import {
+  encodeGuaranteeData,
+  MAGIC_VALUE_DENOTING_A_GUARANTEE,
+} from "../ts/nitro-types";
+import { Exit } from "../ts/types";
+const { ethers } = require("hardhat");
+
+describe("claim (typescript)", function () {
+  const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+  const A_ADDRESS = "0x96f7123E3A80C9813eF50213ADEd0e4511CB820f";
+  const B_ADDRESS = "0x53484E75151D07FfD885159d4CF014B874cd2810";
+
+  it("Can claim with no exit requests", async function () {
+    const initialOutcome: Exit = [
+      {
+        asset: ZERO_ADDRESS,
+        data: "0x",
+        allocations: [
+          {
+            destination: A_ADDRESS,
+            amount: "0x05",
+            callTo: ZERO_ADDRESS,
+            data: "0x",
+          },
+          {
+            destination: B_ADDRESS,
+            amount: "0x05",
+            callTo: ZERO_ADDRESS,
+            data: "0x",
+          },
+        ],
+      },
+    ];
+
+    const guarantee: Exit = [
+      {
+        asset: ZERO_ADDRESS,
+        data: "0x",
+        allocations: [
+          {
+            destination: ZERO_ADDRESS, // TODO: What should this be?
```

Comment:
See answer to your other comment. It should be the address of the target channel, but we may not see the connection between the guarantor channel and the target channel in the pure function you have added (it would probably be in a non-pure wrapping function).